theia-plugin/README.md

```diff
@@ -0,0 +1,2 @@
+# theia-plugin
```

Comment:
Instructions on how to get started would be nice. How do I install all necessary dependencies, how can I start the project?

Reply:
Yes, please, even if it's not published on npm or anything. Just think of a new developer joining the project: ideally, he should be capable of getting the examples running without any help.

Reply:
Done!